web/demos/button/button-active.dart

```diff
@@ -0,0 +1,12 @@
+part of over_react.web.demos;
```

Comment:
I was more just talking about the example methods, which don't have doc comments. But I am fine with leaving it like this.